cybergis_compute_client/Job.py

```diff
@@ -191,13 +263,26 @@ def result_folder_content(self):
         return out
 
     def download_result_folder(self, localPath=None, remotePath=None, raw=False):
+        """
+        Downloads the folder with results from the job using Globus
+        args:
```

Comment:
This looks good, but please capitalize all of the args/returns/raises for consistency. Thanks!